docs/collisions.md

```diff
@@ -64,6 +64,62 @@ For example if a folder has five 5 with the same name say **_A.txt _** the five
 * [Extensions in folder names matter](https://apple.stackexchange.com/questions/123001/renamed-folder-becomes-a-file-with-an-extension) and should be handled :
 * Case sensitivity is taken into account. `Myfile` and `myFile` are different for example. That is useful to [ensure complete syncs](https://rclone.org/overview/#case-insensitive).
 
+#### Extensions
+
+For both files and folders the extension is any string that follows after the last dot.
```

Comment:
I was going to ask if this means `uploadFileName` or in `displayName` or somewhere else, but I see you answer that later.  You might want to put a parenthetical aside here, saying that you're going to address that question below, since it's something the reader will wonder about right away.

Reply:
True! Resolved.

> For both files and folders the extension is any string that follows after the last dot in the `displayName` or `uploadFileName` (see; Source of extensions section below).